lab-max/app/index.html

```diff
@@ -0,0 +1,76 @@
+<!DOCTYPE html>
+<html ng-app="cowsayApp">
+  <head>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <title> Cowsay-Angular Lab </title>
+  </head>
+  <body>
+    <main ng-controller="CowController as cowCtrl" ng-init="cowCtrl.activeCow = 'default'">
+      <section>
+        <h1> {{cowCtrl.title}} </h1>
+        <h3> {{cowCtrl.intro}} </h3>
+        <form
+        novalidate
+        name="cowForm"
+        ng-submit="cowCtrl.cowPopulate(cowCtrl.text)"
+        ng-class="{'app-pink': cowForm.$invalid}"
+        >
+          <input
+          class="form-control"
+          placeholder="Say something!"
+          type="text"
+          ng-model="cowCtrl.text"
+          required ng-minlength="5"
+          ng-maxlength="15"
+          >
+          <select
+          ng-class="{'app-yellow': cowForm.$valid}"
+          ng-model="cowCtrl.activeCow"
+          >
+            <option
+            ng-repeat="cow in cowCtrl.cowChars"
+            value="{{cow}}"
+            >
+              {{cow}}
+            </option>
+          </select>
+          <button
+          class="btn btn-default"
+          ng-class="{'app-green': cowForm.$valid}"
+          type="submit"
+          > {{cowCtrl.submitText}} </button>
+          <button
+          class="btn btn-default"
+          ng-class="{'app-red': cowForm.$valid}"
+          type="button"
+          ng-click="cowCtrl.revertText()"
+          > {{cowCtrl.undoText}} </button>
+        </form>
+        <pre>
+          <p>
+            {{cowCtrl.cowTalk(cowCtrl.text)}}
+          </p>
+        </pre>
+      </section>
+      <section>
+        <i ng-if="!cowCtrl.newText" class="fa fa-moon-o"></i>
+        <i ng-if="!cowCtrl.newText" class="fa fa-angle-double-left"></i>
+        <i ng-if="!cowCtrl.newText" class="fa fa-hand-spock-o"></i>
+        <i ng-if="!cowCtrl.newText" class="fa fa-angle-double-right"></i>
+        <i ng-if="!cowCtrl.newText" class="fa fa-moon-o"></i>
+        <pre>
+          <p
+          ng-class="{'app-circle': cowCtrl.newText}"
+          ng-show="cowCtrl.newText"
+          >
+            {{cowCtrl.newText}}
+          </p>
+          <p ng-hide="cowCtrl.newText">
+            {{cowCtrl.secondCowText}}
+          </p>
+        </pre>
+      </section>
+    </main>
+  </body>
+</html>
```

Comment:
@meeksfred overall, the view looks good - just needs minor cleanup with line spacing/indentation